apisix/cli/ngx_tpl.lua

```diff
@@ -498,6 +498,7 @@ http {
                 return 200;
             }
 
+            proxy_set_header   Host       $upstream_host;
```

Comment:
The spaces between words here is weird.

Reply:
Consider keeping it aligned with the line below. What do you think? Do we need to adjust?

Reply:
Would be better to be aligned. It looks tidy.

Reply:
> Would be better to be aligned. It looks tidy.

agree